tools/tidy/src/tidy.cpp

```diff
@@ -144,28 +200,51 @@ int main(int argc, char** argv) {
     // Check all enabled checks
     for (const auto& checkName : Registry::getEnabledChecks()) {
         const auto check = Registry::create(checkName);
-        OS::print(fmt::format("[{}]", check->name()));
 
-        driver.diagEngine.setMessage(check->diagCode(), check->diagString());
+        if (!quiet)
```

Comment:
Note that for all of these std::optional<bool> arguments you're checking whether they've been provided, not whether they are set to true. Someone could still do something like `--quiet=false` and the tool would instead treat it like quiet was set to true. You actually want `.value_or(false)` or `quiet != true` or something like that.

Reply:
Didn't know about the `--option=value` trick. Fixed :D